index.html

```diff
@@ -114,9 +114,12 @@ <h2 class="h1">Our Clients</h2>
                     <img src="gfx/egmont.svg" alt="Egmont">
                 </div>
                 <div class="c-client__description">
-                  <p>We cooperate with Egmont publishing since a year. Together we released <a href="https://talk-town.dk" target="_blank">Talk Town</a> podcast application.
-                        During this project an iOS, Android, web application and CMS was created.
-                    </p>
+                    <blockquote>
+                        <p>Nephilim is one of our most trusted technological partners. They deliver and develop solutions across the whole stack. They always deliver on time, in great quality and we are always getting a lot of constructive suggestions on complex solutions from them. Over the past year we have, among other things, developed two native mobile apps and built strong business relationship at the same time.
+                            We feel safe having such skilful and trustworthy partner at hand.</p>
+
+                        <footer>- Mads, Head of Digital Development</footer>
```

Comment:
Replace `-` with `–`

```suggestion
                        <footer>– Mads, Head of Digital Development</footer>
```